{{cookiecutter.github_repository}}/requirements/common.txt

```diff
@@ -26,7 +26,7 @@ pytz==2019.1
 # -------------------------------------
 psycopg2-binary==2.8.2
 
-Pillow==6.0.0
+Pillow==5.4.1
```

Comment:
Why have we downgraded this dependency ?

Reply:
There was a conflicting dependency that required Pillow version less than `6.0.0`. In order to combat this issue, this dependency is downgraded.

Reply:
The issue was with django-versatileimagefield.

<img width="1030" alt="Screenshot 2019-05-30 at 11 54 40 AM" src="https://user-images.githubusercontent.com/8039608/58612897-c23c0580-82d1-11e9-8455-862e130841ec.png">


https://travis-ci.com/Fueled/django-init/builds/113651851

Reply:
Djnago versatile image field has add support for pillow 6.x so i think if we should be good updating django-versatileimage in master https://django-versatileimagefield.readthedocs.io/en/latest/#id1

Reply:
@theskumar I see that they've mentioned that in changelog, but the error we got is for the same version: `1.10`: https://github.com/Fueled/django-init/blob/master/%7B%7Bcookiecutter.github_repository%7D%7D/requirements/common.txt#L32

It's already on master and this branch too. Still, Travis and CircleCI were giving the same error.

Reply:
It seems they didn't update the setup.py which still says pillow<6.0 ;?